tests/test-rest-comments-controller.php

```diff
@@ -388,6 +391,96 @@ public function test_create_comment_other_user() {
 		$this->assertEquals( 0, $data['author'] );
 	}
 
+	public function test_create_comment_other_user_without_permission() {
+		wp_set_current_user( $this->subscriber_id );
+
+		$params = array(
+			'post'         => $this->post_id,
+			'author_name'  => 'Homer Jay Simpson',
+			'author_email' => 'chunkylover53@aol.com',
+			'author_url'   => 'http://compuglobalhypermeganet.com',
+			'content'      => 'Here’s to alcohol: the cause of, and solution to, all of life’s problems.',
```

Comment:
Note the smart quotes here in the string, might cause problems, since it relies on editors using UTF-8/etc. Can we use normal ' or escape sequences please? :)

(Also, they have the internet on computers now?)